nnvm/tests/python/frontend/darknet/test_forward.py

```diff
@@ -102,7 +102,7 @@ def test_forward_extraction():
     model_name = 'extraction'
     cfg_name = model_name + '.cfg'
     weights_name = model_name + '.weights'
-    cfg_url = 'https://github.com/pjreddie/darknet/blob/master/cfg/' + cfg_name + '?raw=true'
+    cfg_url = 'https://github.com/siju-samuel/darknet/blob/master/cfg/' + cfg_name + '?raw=true'

```

Comment:
any reason why not keep upstream with darknet?

Reply:
yolo.cfg is not available any more on the darknet/upstream anymore, because of that moved to forked one.

Reply:
moving things into a forked one is quite slippery. I would try to not do that as much as possible and instead focus on the cfgs that are supported in the upstream

Reply:
the single layer test looks fine

Reply:
@PariksheetPinjari909  act on the comment?